llm_on_ray/inference/serve.py

```diff
@@ -187,9 +130,7 @@ def main(argv=None):
         # all models are served under the same URL and then accessed
         # through model_id, so it needs to pass in a unified URL.
         host = "127.0.0.1" if args.serve_local_only else "0.0.0.0"
-        rp = args.route_prefix if args.route_prefix else ""
-        route_prefix = "/{}".format(rp)
-        openai_serve_run(deployments, host, route_prefix, args.port, args.max_concurrent_queries)
+        openai_serve_run(deployments, host, "/", 8000, args.max_concurrent_queries)
```

Comment:
we can't use a fixed port here as the previous port is configurable.

Reply:
revised